### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by ipberry**
## Description
Updating Docs

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] added defaults to input generator
  - [x] adding the input generator to the nav bar
  - [x] updating github dark mode logo 

## Questions
- [x] setting the colors for dark mode for github and the website seem to work in all of my browsers except chrome (maybe we look into that later)

## Status
- [x] Ready to go

### CODE REVIEW

File: docs/input_generator.rst [hunk 1]
```diff
@@ -166,30 +166,30 @@ Input Generator
            </label>
            <div id="nwchem-options" style="display: none; margin-left: 20px;">
                <label for="nwchem_scratch">NWChem Scratch:</label>
-               <input type="text" id="nwchem_scratch" placeholder="Enter NWChem scratch directory" style="margin-bottom: 10px;"><br>
+               <input type="text" id="nwchem_scratch" placeholder="None" style="margin-bottom: 10px;"><br>
                
                <label for="nwchem_perm">NWChem Perm:</label>
```
Comment (carolinesargent) [inline docs/input_generator.rst:171]: NWChem Permanent Directory

File: docs/input_generator.rst [hunk 2]
```diff
@@ -166,30 +166,30 @@ Input Generator
            </label>
            <div id="nwchem-options" style="display: none; margin-left: 20px;">
                <label for="nwchem_scratch">NWChem Scratch:</label>
```
Comment (carolinesargent) [inline docs/input_generator.rst:168]: NWChem Scratch Directory

File: docs/input_generator.rst [hunk 3]
```diff
@@ -166,30 +166,30 @@ Input Generator
...
-               <input type="text" id="nwchem_scratch" placeholder="Enter NWChem scratch directory" style="margin-bottom: 10px;"><br>
+               <input type="text" id="nwchem_scratch" placeholder="None" style="margin-bottom: 10px;"><br>
                
                <label for="nwchem_perm">NWChem Perm:</label>
-               <input type="text" id="nwchem_perm" placeholder="Enter NWChem perm directory" style="margin-bottom: 10px;"><br>
+               <input type="text" id="nwchem_perm" placeholder="Default: None" style="margin-bottom: 10px;"><br>
                
                <label for="nwchem_scf">NWChem SCF:</label>
```
Comment (carolinesargent) [inline docs/input_generator.rst:174]: NWChem SCF Options, also should we specify that this needs to be a dictionary?

File: docs/input_generator.rst [hunk 4]
```diff
@@ -166,30 +166,30 @@ Input Generator
...
-               <input type="text" id="nwchem_scf" placeholder="Enter SCF options" style="margin-bottom: 10px;"><br>
+               <input type="text" id="nwchem_scf" placeholder="Default: None" style="margin-bottom: 10px;"><br>
                
                <label for="nwchem_dft">NWChem DFT:</label>
```
Comment (carolinesargent) [inline docs/input_generator.rst:177]: NWChem DFT Options, also will need to be a dict

File: docs/input_generator.rst [hunk 5]
```diff
@@ -166,30 +166,30 @@ Input Generator
...
        
            <label style="display: block;">
                <input type="checkbox" id="software_qchem" name="software" onclick="toggleSoftware('qchem')"> Q-Chem
            </label>
            <div id="qchem-options" style="display: none; margin-left: 20px;">
                <label for="qchem_options">Q-Chem Options:</label>
-               <input type="text" id="qchem_options" placeholder="Enter Q-Chem options" style="margin-bottom: 10px;"><br>
+               <input type="text" size = "25" id="qchem_options" placeholder="Default: {'JOBTYPE': 'xsapt or sp'}" style="margin-bottom: 10px;"><br>
                
                <label for="qchem_sapt">Q-Chem SAPT:</label>
```
Comment (carolinesargent) [inline docs/input_generator.rst:191]: Q-Chem SAPT Options

File: docs/input_generator.rst (hunk 1 above)
Comment (ipberry) [reply docs/input_generator.rst]: updated these! thanks!

File: docs/input_generator.rst [hunk 6]
```diff
@@ -198,7 +198,7 @@ Input Generator
            <div id="psi4-options" style="display: none; margin-left: 20px;">
                <label><input type="checkbox" id="fisapt_partition" style="margin-bottom: 10px;"> FISAPT Partition</label><br>
                <label><input type="checkbox" id="do_fsapt" style="margin-bottom: 10px;"> Do FSAPT</label><br>
-               <label for="psi4options">Psi4 Options:</label>
+               <label for="psi4options">Additional Options Dcitionary:</label>
```
Comment (carolinesargent) [inline docs/input_generator.rst:201]: spelling of dictionary

**Pull request merged**